README.md

```diff
@@ -40,15 +52,91 @@ or
 python setup.py develop
 ```
 
+## Getting started
+`s2spy` provides end-to-end solutions for machine learning (ML) based S2S forecasting.
+
+![workflow](./docs/assets/images/workflow.png)
+
+### Datetime operations & Data processing
+In a typical ML-based S2S project, the first step is always data processing.  A calendar-based datetime operation module is implemented to help the users prepare their data. For instance, if a user is looking for predictors for winter climate at seasonal timescales (~180 days), a `AdventCalendar` can be used to prepare the data:
+
+```py
+calendar = s2spy.time.AdventCalendar(anchor=(11, 30), freq='180d')
+calendar = calendar.map_years(2020, 2021)
+calendar
+>>>    i_interval                 (target) 0                         1
+>>>    anchor_year
+>>>    2021         (2021-06-03, 2021-11-30]  (2020-12-05, 2021-06-03]
+>>>    2020         (2020-06-03, 2020-11-30]  (2019-12-06, 2020-06-03]
+```
+
+`s2spy` works with `xarray` data. Now, the user can load their data and easily resample their data to the desired timescales configured in the calendar:
```

Comment:
```suggestion
`s2spy` works with `xarray` data. Now, the user can load the data `input_data` and resample it to the desired timescales configured in the calendar:
```

Reply:
another comment on **`s2spy` works with `xarray` data**. It works also with `pandas dataframe`, right? the `bins` output shown below are produced when the `input_data` is a dataframe.

Reply:
Yes, this is correct for now. But since we are implementing more features which require the support of multi-dimensional data, we plan to drop pandas for new features (e.g. dimensionality reduction) and therefore it is better to make it clear that we support xarray. Only the calendar could somehow work with pandas, but it is not recommended. To avoid confusion, I think it is better we only mention xarray.